plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -36,25 +34,33 @@ def quadratic(cfg: DictConfig) -> Any:
     return z
 
 
-# TODO: try to enable this
-# # Many sweepers are batching jobs in groups.
-# # This test suite verifies that the spawned jobs are not overstepping the directories of one another.
-# @pytest.mark.parametrize(
-#     "launcher_name, overrides",
-#     [
-#         (
-#             "basic",
-#             [
-#                 "hydra/sweeper=ax",
-#                 # This will cause the sweeper to split batches to at most 2 jobs each, which is what
-#                 # the tests in BatchedSweeperTestSuite are expecting.
-#                 "hydra.sweeper.params.max_batch_size=2",
-#             ],
-#         )
-#     ],
-# )
-# class TestExampleSweeperWithBatching(BatchedSweeperTestSuite):
-#     ...
+def test_jobs_dirs(sweep_runner: TSweepRunner) -> None:
+    # Verify that the spawned jobs are not overstepping the directories of one another.
+    sweep = sweep_runner(
+        calling_file=os.path.dirname(os.path.abspath(__file__)),
```

Comment:
calling file is used to detect the directory of the config.
this can be much more explicit (fix other instances of this in the tests too).
example from the main tests:
```python
    with sweep_runner(
        calling_file="examples/tutorial/5_composition/my_app.py",
```